examples/contrib/export_har/exporthar.py

```diff
@@ -0,0 +1,199 @@
+import base64
+import json
+import logging
+from datetime import datetime
+from datetime import timezone
+
+from mitmproxy import command
+from mitmproxy import version
+from mitmproxy.net.http import cookies
+from mitmproxy.utils import strutils
+
+
+logger = logging.getLogger(__name__)
+
+
+class ExportHar:
+    def __init__(self):
+        self.SERVERS_SEEN = set()
```

Comment:
Short explanation what this variable is used for would be handy. Also it should probably be lowercase given that it's a regular attribute. :)